api/api_test.go

```diff
@@ -1170,6 +1170,8 @@ func TestServer_SendAction(t *testing.T) {
 	}}
 
 	chain.EXPECT().ChainID().Return(uint32(1)).Times(2)
+	chain.EXPECT().TipHeight().Return(uint64(4)).Times(2)
+	svr.cfg.Genesis.KamchatkaBlockHeight = 10
```

Comment:
don't need this line, since it is not tested in api_test?

Reply:
#https://github.com/iotexproject/iotex-core/pull/2777/files#diff-bf98d5fab5bcfabded9769069e7318683f549ec91cff84c5f87a9ae3a3ca3d8dR365
if not, got an error.
```
        	Error Trace:	api_test.go:1180
        	Error:      	Received unexpected error:
        	            	rpc error: code = InvalidArgument desc = ChainID does not match, expecting 1, got 0
        	Test:       	TestServer_SendAction
```